code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -270,10 +270,27 @@
 /obj/item/weapon/robot_module/butler/New()
 	..()
 	modules += new /obj/item/weapon/reagent_containers/food/drinks/cans/beer(src)
+	modules += new /obj/item/weapon/reagent_containers/food/drinks/cans/cola(src)
+	modules += new /obj/item/weapon/reagent_containers/food/drinks/cans/sodawater(src)
 	modules += new /obj/item/weapon/reagent_containers/food/condiment/enzyme(src)
+	modules += new /obj/item/weapon/reagent_containers/food/drinks/bottle/orangejuice(src) // -0.3 oxy/sec
+	modules += new /obj/item/weapon/reagent_containers/food/drinks/bottle/tomatojuice(src) // -0.2 fire/sec
+	modules += new /obj/item/weapon/reagent_containers/food/drinks/bottle/limejuice(src) // -0.2 tox/sec
+	modules += new /obj/item/weapon/reagent_containers/food/drinks/coffee(src) // -1 paralysis stunned & weakened/sec
+	modules += new /obj/item/weapon/reagent_containers/food/drinks/tea(src)
+	modules += new /obj/item/weapon/reagent_containers/food/drinks/bottle/milk(src) // -0.2 brute/sec
+	modules += new /obj/item/weapon/reagent_containers/food/condiment/sugar(src)
+	modules += new /obj/item/weapon/reagent_containers/food/drinks/ice(src)
+	modules += new /obj/item/weapon/reagent_containers/food/drinks/bottle/cream(src)
+
+	modules += new /obj/item/weapon/reagent_containers/food/drinks/bottle/tequila(src)
+	modules += new /obj/item/weapon/reagent_containers/food/drinks/bottle/vodka(src)
+	modules += new /obj/item/weapon/reagent_containers/food/drinks/bottle/whiskey(src)

```

Comment:
Ah, how will all these interact with borg rechargers?

Reply:
Not sure what you mean? They drain your cell to recharge, so they're effectively infinite so long as you recharge occasionally. Is that what you meant?

Reply:
You currently do not handle refilling these anywhere is what he is really pointing out. The enzyme and emagged beer bottle are being specially handled in the respawn_consumable proc below (oddly the normal beer bottle isn't). When these bottles run out, they won't refill from a sit in a recharger.

Interestingly, the rechargers seem to be calling these same bits of code it its process proc and then calling respawn_consumables (effectively doubling up the refilling).

Reply:
When I tested, they seemed to refill automatically. Will retest and be sure.

Reply:
I retested it to be sure. The reagent containers auto-refill already, without any further additions.

Reply:
https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/food_and_drinks/drinks/drinks.dm#L98-L109
:fire: :fire: :eyes: :fire: :fire: